src/main/webapp/resources/js/adminJs/adm-discount.js

```diff
@@ -77,8 +77,10 @@ function addNewDiscount(method,bool){
   var inputData = {
     reason: $("#DReason").val(),
     value : $("#DValue").val(),
-    startDate :startDate,
-    endDate : endDate,
+    startDate : "23-03-2017",
+    endDate : "23-03-2017",
+    startTime: "11:00",
+    endTime: "23:00",

```

Comment:
It's just for the test. You don't have to write smth like "Magic numbers, please remove it". I will remove it all  when Yevgen will finish JQuery tables